docs/hub/how-to-downstream.md

```diff
@@ -0,0 +1,51 @@
+# How to integrate downstream methods in your library
+
+In the tutorial, you learned some basic functions for integrating the Hub into your library. Functions that allow your library to download files from the Hub are referred to as *downstream* functions. This guide introduces additional downstream methods you can integrate in your library. You will learn how to:
```

Comment:
In which tutorial?

Reply:
The tutorial refers to `tutorial-add-library` which is intended to provide a general and more approachable introduction to the entire integration process. The `how-to-downstream` and `how-to-upstream` guides digs a little deeper into each specific part of the integration, and presents all the other alternative options.

You actually bring up a really good point about methods vs. functions, and we should definitely keep it consistent. I was struggling a bit with the language here, and I'm not sure which word is better. What do you think is more accurate?

Reply:
Ok I settled on **utilities** since **methods** and **functions** both refer to specific things in Python.